modules/nf-core/lofreq/indelqual/tests/main.nf.test

```diff
@@ -0,0 +1,64 @@
+nextflow_process {
+
+    name "Test Process LOFREQ_INDELQUAL"
+    script "../main.nf"
+    process "LOFREQ_INDELQUAL"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "lofreq"
+    tag "lofreq/indelqual"
+
+    test("sarscov2 - bam, fasta") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test', single_end:false ], // meta map
+                    file(params.test_data['sarscov2']['illumina']['test_paired_end_bam'], checkIfExists: true)
+                    ]
+                input[1] = [
+                        file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true)
+                    ]
```

Comment:
```suggestion
                input[0] = [  [ id:'test', single_end:false ], // meta map
                    file(params.test_data['sarscov2']['illumina']['test_paired_end_bam'], checkIfExists: true)
                    ]
                input[1] = file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true)
```
Can you update the paths as described here: https://nf-co.re/docs/guidelines/components/modules#input-data